services/order.py

```diff
@@ -0,0 +1,27 @@
+from django.db import transaction
+from db.models import Order, Ticket, User
+from django.db.models import QuerySet
+
+
+@transaction.atomic
+def create_order(tickets: list[dict],
+                 username: str,
+                 date: str = None) -> Order:
```

Comment:
The `date` parameter is expected to be a string, but it would be more appropriate to use a `datetime` object for the `created_at` field. This ensures better validation and manipulation of date and time data.